### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by tonyfruzza**
* additional message about pygamezero

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # python-class
 Projects I did in class of 2023
+
+Here is the note that your father added. We later worked on some [PyGameZero](https://pygame-zero.readthedocs.io/en/stable/) projects.
```
Comment (dylan-fruzza) [inline README.md]: do we need to say who added this?

**Review by dylan-fruzza — CHANGES_REQUESTED** [review]
see my note

File: README.md (hunk 1 above)
Comment (tonyfruzza) [reply README.md]: Okay thank you I've made a commit with your consideration

Comment (dylan-fruzza) [reply README.md]: thanks

**Pull request merged**